metashare/api/jobs.py

```diff
@@ -98,22 +93,26 @@ def _create_org_and_run_flow(scratch_org, *, user, repo_url, repo_branch, projec
     )
 
 
-def create_branches_on_github_then_create_scratch_org(
-    *, project, repo_url, scratch_org, task, user
-):
+def create_branches_on_github_then_create_scratch_org(*, scratch_org):
+    scratch_org.refresh_from_db()
+    user = scratch_org.owner
+    task = scratch_org.task
+    project = task.project
```

Comment:
Maybe this is just a stylistic preference, or maybe you have good reasons to prefer otherwise -- but it seems odd to be to pass in a long list of args when all except one can be derived?

Reply:
This was driven by TDD considerations, where it was easier to pass them in separately if I wanted to tweak certain values. I'm OK with bundling it up at this point in the process though.